__tests__/index.test.js

```diff
@@ -180,3 +180,41 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("status 201: responds with the posted comment object", () => {
+      return request(app)
+          .post("/api/articles/1/comments")
+          .send({username: "butter_bridge", body: "This is a test comment"})
+          .expect(201)
+          .then(({body}) => {
+              expect(body.comment_id).toBe(19);
+              expect(body.author).toBe("butter_bridge");
+              expect(body.article_id).toBe(1);
+              expect(body.body).toBe("This is a test comment");
+          });
+  });
```

Comment:
Please add another test where unnecessary properties in the post object are ignored. For example when posting with this:

`{username: "butter_bridge", body: "This is a test comment", extraProperty:"extraValue"}`